cmd/policy-main.go

```diff
@@ -395,18 +419,23 @@ func runPolicyLinksCmd(args cli.Args, recursive bool) {
 func runPolicyCmd(args cli.Args) {
 	var operation, policyStr string
 	var probeErr *probe.Error
-	perms := accessPerms(args.Get(0))
-	targetURL := args.Get(1)
+	perms := accessPerms(args.Get(1))
+	targetURL := args.Get(2)
 	if perms.isValidAccessPERM() {
 		probeErr = doSetAccess(targetURL, perms)
 		operation = "set"
 	} else if perms.isValidAccessFile() {
 		probeErr = doSetAccessJSON(targetURL, perms)
 		operation = "setJSON"
 	} else {
-		targetURL = args.First()
-		perms, policyStr, probeErr = doGetAccess(targetURL)
+		targetURL = args.Get(1)
 		operation = "get"
+		if args.First() == "get-json" {
+			operation = "get-json"
```

Comment:
The operation name can be camel cased as well - `getJSON`. this will be similar to the other operation names - `set`,`setJSON`

Reply:
Good catch @Praveenrajmani  :+1:  `operation` is being used in the error message so changed it to `set json` and `get json`  which gives a clear info in error message compared to `setJSON` which was used earlier.